Author: fuankg
Date: Mon Jan 19 00:35:25 2015
New Revision: 1652886

URL: http://svn.apache.org/r1652886
Log:
Enable to build mod_lua against Lua 5.3.

Modified:
    httpd/httpd/trunk/modules/lua/NWGNUmakefile
    httpd/httpd/trunk/modules/lua/mod_lua.h

Modified: httpd/httpd/trunk/modules/lua/NWGNUmakefile
URL: http://svn.apache.org/viewvc/httpd/httpd/trunk/modules/lua/NWGNUmakefile?rev=1652886&r1=1652885&r2=1652886&view=diff
==============================================================================
--- httpd/httpd/trunk/modules/lua/NWGNUmakefile (original)
+++ httpd/httpd/trunk/modules/lua/NWGNUmakefile Mon Jan 19 00:35:25 2015
@@ -46,6 +46,9 @@ XCFLAGS		+= \
 #
 XDEFINES	+= \
 			-DLUA_COMPAT_ALL \
+			-DLUA_COMPAT_5_2 \
+			-DLUA_COMPAT_5_1 \
+			-DLUA_COMPAT_MODULE \
 			$(EOLIST)
 
 #

Modified: httpd/httpd/trunk/modules/lua/mod_lua.h
URL: http://svn.apache.org/viewvc/httpd/httpd/trunk/modules/lua/mod_lua.h?rev=1652886&r1=1652885&r2=1652886&view=diff
==============================================================================
--- httpd/httpd/trunk/modules/lua/mod_lua.h (original)
+++ httpd/httpd/trunk/modules/lua/mod_lua.h Mon Jan 19 00:35:25 2015
@@ -43,6 +43,10 @@
 
 /* Allow for Lua 5.2 backwards compatibility */
 #define LUA_COMPAT_ALL
+/* Allow for Lua 5.3 backwards compatibility */
+#define LUA_COMPAT_5_2
+#define LUA_COMPAT_5_1
+#define LUA_COMPAT_MODULE
 
 #include "lua.h"
 #include "lauxlib.h"
